frontend/src/components/TodoTask/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from 'antd';
+import { StarFilled, StarOutlined, BorderOutlined, CheckSquareOutlined } from '@ant-design/icons';
+import { blue } from '@ant-design/colors';
+
+export type IconButtonProps = {
+  onClick(): void;
+  color?: string;
+  variant: 'star' | 'check';
+  state: boolean;
+};
+
+export const IconButton = ({ onClick, color = blue.primary, state, variant }: IconButtonProps) => {
+  const onIcon =
+    variant === 'star' ? (
+      <StarFilled style={{ color: color }} />
```

Comment:
O tym rozmawialiśmy, to nie zapomnij poprawić :)

Reply:
done